Hotfix content updates

StacSch opened the pull request:
Updates to Help, need to go on Staging for Greg
@kaitlin 
@kristine.sweet
@nmonga91

kaitlin commented on src/help/changelog.md:
```diff
@@ -1,35 +1,37 @@
 
- Users can now add additional columns to their submission files (A-C) that will be returned in their warning and error files. To use this feature, add any column to your submission data and prefix the header with "flex\_". For example, a column named "flex_reportingbureau" will be ignored for validation purposes but returned for any rows that have errors in the warnings and error reports. 
+Users will be able to generate D files outside the context of a submission. In other words, they won't need to validate A-C to be able to generate D files.
 
```
We should add the link here to point agencies to the page where they can do this.

kaitlin commented on src/help/validations.csv:
```diff
@@ -3,47 +3,47 @@ A/B/C,A1,"TAS components: The combination of all the elements that make up the T
-A,A12,AdjustmentsToUnobligatedBalanceBroughtForward_CPE= aggregate value for GTAS SF 133 line #1010 through 1042 for the same reporting period.,Fatal Error,Implemented,Calculation
-A,A14,GrossOutlayAmountByTAS_CPE= value for GTAS SF 133 line #3020 for the same reporting period.,Fatal Error,Implemented,Calculation
-A,A15,UnobligatedBalance_CPE= value for GTAS SF 133 line #2490 for the same reporting period.,Fatal Error,Implemented,Calculation
+A,A6,"BudgetAuthorityAvailableAmountTotal_CPE= CPE value for GTAS SF 133 line #1910, for the same reporting period","Warning currently, Fatal Error in future release",Implemented,Calculation
```
it looks like this is reverting earlier changes. These should have been changed to errors but it looks like this PR is reverting it back to "Warning currently, error in future release"

StacSch commented:
I’ll take care of that.

Stacia

From: Kaitlin Devine [mailto:notifications@github.com]
Sent: Friday, December 16, 2016 12:01 PM
To: fedspendingtransparency/data-act-broker-web-app <data-act-broker-web-app@noreply.github.com>
Cc: Stacia Schwartz <stacia.schwartz@kearneyco.com>; Author <author@noreply.github.com>
Subject: Re: [fedspendingtransparency/data-act-broker-web-app] Hotfix content updates (#404)


@kaitlin commented on this pull request.

________________________________

In src/help/validations.csv<https://github.com/fedspendingtransparency/data-act-broker-web-app/pull/404#pullrequestreview-13371169>:

> @@ -3,47 +3,47 @@ A/B/C,A1,"TAS components: The combination of all the elements that make up the T

 A,A2,BudgetAuthorityAvailableAmountTotal_CPE = BudgetAuthorityAppropriatedAmount_CPE + BudgetAuthorityUnobligatedBalanceBroughtForward_FYB + AdjustmentsToUnobligatedBalanceBroughtForward_CPE + OtherBudgetaryResourcesAmount_CPE,Fatal Error,Implemented,Calculation

 A,A3,OtherBudgetaryResourcesAmount_CPE = ContractAuthorityAmountTotal_CPE + BorrowingAuthorityAmountTotal_CPE + SpendingAuthorityfromOffsettingCollectionsAmountTotal_CPE,Fatal Error,Implemented,Calculation

 A,A4,StatusOfBudgetaryResourcesTotal_CPE= ObligationsIncurredTotalByTAS_CPE + UnobligatedBalance_CPE,Fatal Error,Implemented,Calculation

-A,A6,"BudgetAuthorityAvailableAmountTotal_CPE= CPE value for GTAS SF 133 line #1910, for the same reporting period",Fatal Error,Implemented,Calculation

-A,A7,"BudgetAuthorityUnobligatedBalanceBroughtForward_FYB= value for GTAS SF 133 line #1000, for the same reporting period",Fatal Error,Implemented,Calculation

-A,A8,"BudgetAuthorityAppropriatedAmount_CPE= CPE aggregate value for GTAS SF 133 line #1160 + #1180 + #1260 + #1280, for the same reporting period",Fatal Error,Implemented,Calculation

-A,A9,"ContractAuthorityAmountTotal_CPE= CPE aggregate value for GTAS SF 133 line #1540 + #1640, for the same reporting period",Fatal Error,Implemented,Calculation

-A,A10,BorrowingAuthorityAmountTotal_CPE= aggregate value for GTAS SF 133 line #1340 + #1440 for the same reporting period.,Fatal Error,Implemented,Calculation

-A,A11,SpendingAuthorityfromOffsettingCollectionsAmountTotal_CPE= aggregate value for GTAS SF 133 line #1750 + #1850 for the same reporting period.,Fatal Error,Implemented,Calculation

-A,A12,AdjustmentsToUnobligatedBalanceBroughtForward_CPE= aggregate value for GTAS SF 133 line #1010 through 1042 for the same reporting period.,Fatal Error,Implemented,Calculation

-A,A14,GrossOutlayAmountByTAS_CPE= value for GTAS SF 133 line #3020 for the same reporting period.,Fatal Error,Implemented,Calculation

-A,A15,UnobligatedBalance_CPE= value for GTAS SF 133 line #2490 for the same reporting period.,Fatal Error,Implemented,Calculation

+A,A6,"BudgetAuthorityAvailableAmountTotal_CPE= CPE value for GTAS SF 133 line #1910, for the same reporting period","Warning currently, Fatal Error in future release",Implemented,Calculation

it looks like this is reverting earlier changes. These should have been changed to errors but it looks like this PR is reverting it back to "Warning currently, error in future release"

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/fedspendingtransparency/data-act-broker-web-app/pull/404#pullrequestreview-13371169>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AV1VjJzPlzaiiICoMnvpJ3lkorntLY-Uks5rIsPCgaJpZM4LPWSX>.

________________________________
This e-mail is intended solely for the person or entity to which it is addressed and may contain confidential and/or privileged information. Any review, dissemination, copying, printing or other use of this e-mail by persons or entities other than the addressee is prohibited. If you have received this e-mail in error, please contact the sender immediately and delete the material from any computer.

kaitlin commented on src/help/validations.csv:
```diff
@@ -3,47 +3,47 @@ A/B/C,A1,"TAS components: The combination of all the elements that make up the T
 B,B3,"ObligationsUndeliveredOrdersUnpaidTotal in File B = USSGL 4801 + 4881 in File B for the same date context (FYB or CPE). This applies to the program activity and object class level. Note for FYB values, only 4801 is expected to have a balance other than zero.",Warning,Implemented,Calculation
 B,B4,"ObligationsDeliveredOrdersUnpaidTotal in File B = USSGL 4901 + 4981 in File B for the same date context (FYB or CPE). This applies to the program activity and object class level. Note for FYB values, only 4901 is expected to have a balance other than zero.",Warning,Implemented,Calculation
-B,B5,GrossOutlayAmountByProgramObjectClass (File B) = GrossOutlaysUndeliveredOrdersPrepaidTotal (File B) + GrossOutlaysDeliveredOrdersPaidTotal (File B) for the same date context (FYB or CPE).,Warning,Implemented,Calculation
+B,B5,GrossOutlayAmountByProgramObjectClass_FYB (File B) = GrossOutlaysUndeliveredOrdersPrepaidTotal_FYB (File B) + GrossOutlaysDeliveredOrdersPaidTotal_FYB (File B). GrossOutlayAmountByProgramObjectClass_CPE (File B) should reflect year-to-date activity as of the end of the reporting period: GrossOutlayAmountByProgramObjectClass_CPE = (GrossOutlaysUndeliveredOrdersPrepaidTotal_CPE less GrossOutla ...
```
I think Kristine needs to review these

StacSch commented:
I need to make changes to two of the files, then put in the PRs.

Stacia

From: Kaitlin Devine [mailto:notifications@github.com]
Sent: Friday, December 16, 2016 12:01 PM
To: fedspendingtransparency/data-act-broker-web-app <data-act-broker-web-app@noreply.github.com>
Cc: Stacia Schwartz <stacia.schwartz@kearneyco.com>; Author <author@noreply.github.com>
Subject: Re: [fedspendingtransparency/data-act-broker-web-app] Hotfix content updates (#404)


@kaitlin commented on this pull request.

________________________________

In src/help/validations.csv<https://github.com/fedspendingtransparency/data-act-broker-web-app/pull/404#pullrequestreview-13371169>:

> @@ -3,47 +3,47 @@ A/B/C,A1,"TAS components: The combination of all the elements that make up the T

 A,A2,BudgetAuthorityAvailableAmountTotal_CPE = BudgetAuthorityAppropriatedAmount_CPE + BudgetAuthorityUnobligatedBalanceBroughtForward_FYB + AdjustmentsToUnobligatedBalanceBroughtForward_CPE + OtherBudgetaryResourcesAmount_CPE,Fatal Error,Implemented,Calculation

 A,A3,OtherBudgetaryResourcesAmount_CPE = ContractAuthorityAmountTotal_CPE + BorrowingAuthorityAmountTotal_CPE + SpendingAuthorityfromOffsettingCollectionsAmountTotal_CPE,Fatal Error,Implemented,Calculation

 A,A4,StatusOfBudgetaryResourcesTotal_CPE= ObligationsIncurredTotalByTAS_CPE + UnobligatedBalance_CPE,Fatal Error,Implemented,Calculation

-A,A6,"BudgetAuthorityAvailableAmountTotal_CPE= CPE value for GTAS SF 133 line #1910, for the same reporting period",Fatal Error,Implemented,Calculation

-A,A7,"BudgetAuthorityUnobligatedBalanceBroughtForward_FYB= value for GTAS SF 133 line #1000, for the same reporting period",Fatal Error,Implemented,Calculation

-A,A8,"BudgetAuthorityAppropriatedAmount_CPE= CPE aggregate value for GTAS SF 133 line #1160 + #1180 + #1260 + #1280, for the same reporting period",Fatal Error,Implemented,Calculation

-A,A9,"ContractAuthorityAmountTotal_CPE= CPE aggregate value for GTAS SF 133 line #1540 + #1640, for the same reporting period",Fatal Error,Implemented,Calculation

-A,A10,BorrowingAuthorityAmountTotal_CPE= aggregate value for GTAS SF 133 line #1340 + #1440 for the same reporting period.,Fatal Error,Implemented,Calculation

-A,A11,SpendingAuthorityfromOffsettingCollectionsAmountTotal_CPE= aggregate value for GTAS SF 133 line #1750 + #1850 for the same reporting period.,Fatal Error,Implemented,Calculation

-A,A12,AdjustmentsToUnobligatedBalanceBroughtForward_CPE= aggregate value for GTAS SF 133 line #1010 through 1042 for the same reporting period.,Fatal Error,Implemented,Calculation

-A,A14,GrossOutlayAmountByTAS_CPE= value for GTAS SF 133 line #3020 for the same reporting period.,Fatal Error,Implemented,Calculation

-A,A15,UnobligatedBalance_CPE= value for GTAS SF 133 line #2490 for the same reporting period.,Fatal Error,Implemented,Calculation

+A,A6,"BudgetAuthorityAvailableAmountTotal_CPE= CPE value for GTAS SF 133 line #1910, for the same reporting period","Warning currently, Fatal Error in future release",Implemented,Calculation

it looks like this is reverting earlier changes. These should have been changed to errors but it looks like this PR is reverting it back to "Warning currently, error in future release"

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/fedspendingtransparency/data-act-broker-web-app/pull/404#pullrequestreview-13371169>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AV1VjJzPlzaiiICoMnvpJ3lkorntLY-Uks5rIsPCgaJpZM4LPWSX>.

________________________________
This e-mail is intended solely for the person or entity to which it is addressed and may contain confidential and/or privileged information. Any review, dissemination, copying, printing or other use of this e-mail by persons or entities other than the addressee is prohibited. If you have received this e-mail in error, please contact the sender immediately and delete the material from any computer.
